config/configuration.go

```diff
@@ -29,14 +28,17 @@ const (
 
 	// Route53MaintenanceWndMinute configuration key for the maintenance window for Route53.
 	// This is the maintenance window before the next AWS Route53 pricing period starts
-	Route53MaintenanceWndMinute = "route53.maintenanceWindowMinute"
+	Route53MaintenanceWndMinute = "route53.mainthelm.banzaiRepositoryURenanceWindowMinute"
```

Comment:
I think this configuration flag has a bit confusing name.

Reply:
Fixed